### PR TITLE
Add Conecto TS0601 `_TZE200_zppcgbdj` temperature & humidity

### LINKED ISSUE #3024: [Device Support Request] TS0601 _TZE200_zppcgbdj (Conecto temperature/humidity with display)
### Problem description

This is a Tuya temperature & humidity sensor with e-ink display sold under the brand name "Conecto" in the Nordics. It is similar to one of the existing quirks, but with a different humidity multiplier.

There are few hits on google for Conecto, so in case this ends up in search results: The Conecto temperature & humidity sensor _without_ e-ink display is already supported, as is the Conecto water leak sensor.

### Solution description

A pull request containing a quirk will be created by me shortly. Doing final testing now.

### Screenshots/Video

_No response_

### Device signature

<details><summary>Device signature</summary>

```json
{
  "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.EndDevice: 2>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress: 128>, manufacturer_code=4417, maximum_buffer_size=66, maximum_incoming_transfer_size=66, server_mask=10752, maximum_outgoing_transfer_size=66, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=True, *is_full_function_device=False, *is_mains_powered=False, *is_receiver_on_when_idle=False, *is_router=False, *is_security_capable=False)",
  "endpoints": {
    "1": {
      "profile_id": "0x0104",
      "device_type": "0x0302",
      "input_clusters": [
        "0x0001",
        "0x0402",
        "0x0405",
        "0xef00"
      ],
      "output_clusters": [
        "0x000a",
        "0x0019"
      ]
    }
  },
  "manufacturer": "_TZE200_zppcgbdj",
  "model": "TS0601",
  "class": "ts0601_sensor.TuyaTempHumiditySensor_Conecto_Display"
}
```

**NOTE:** The `class`  here is _with_ the new quirk that I am currently validating. Without it, there are no matching quirks, and the device will not contain the Tuya temperature+humidity+battery cluster.

</details>

### Diagnostic information

_No response_

### Logs

_No response_

### Custom quirk

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by pianosaurus**
The device is a rebranded Tuya temperature and humidity sensor, and the quirk is almost identical to one of the existing Tuya quirks (but with a different humidity multiplier). I am not sure if the device is also sold under other brand names, though I suppose that is quite likely. I figure it makes sense to put it in as a Tuya quirk.

## Proposed change

This pull request adds the quirk immediately after the most similar existing quirk. It is almost identical, though if follows the standard set by the rest of the file when it comes to humidity multiplier (commented line containing the default value of 100).

## Additional information

Fixes #3024 

I do not have the development environment available to run formatting or adding tests. Given that this is essentially a copy-paste of the quirk before it, and it works in manual testing, I hope that is OK.

## Checklist

- [x] The changes are tested and work correctly
- [ ] `pre-commit` checks pass / the code has been formatted using Black
- [ ] Tests have been added to verify that the new code works

### CODE REVIEW

File: zhaquirks/tuya/ts0601_sensor.py [hunk 1]
```diff
@@ -143,6 +143,53 @@ class TuyaTempHumiditySensor(CustomDevice):
     }
 
 
+class TuyaTempHumiditySensor_Conecto_Display(CustomDevice):
```
Comment (javicalle) [inline zhaquirks/tuya/ts0601_sensor.py]: Don't use brand names to quirks.
Maybe better `TuyaTempHumiditySensorVar02`?

File: zhaquirks/tuya/ts0601_sensor.py [hunk 2]
```diff
@@ -143,6 +143,53 @@ class TuyaTempHumiditySensor(CustomDevice):
...
+    """Custom device representing tuya temp and humidity sensor with e-ink screen."""
+
+    # RelativeHumidity multiplier
+    # RH_MULTIPLIER = 100
```
Comment (javicalle) [inline zhaquirks/tuya/ts0601_sensor.py]: Remove comments

File: zhaquirks/tuya/ts0601_sensor.py [hunk 3]
```diff
@@ -143,6 +143,53 @@ class TuyaTempHumiditySensor(CustomDevice):
...
+
+    signature = {
+        # <SimpleDescriptor endpoint=1, profile=260, device_type=81
+        # device_version=1
+        # input_clusters=[4, 5, 61184, 0]
+        # output_clusters=[25, 10]>
+        MODELS_INFO: [
+            ("_TZE200_zppcgbdj", "TS0601"),
```
Comment (javicalle) [inline zhaquirks/tuya/ts0601_sensor.py]: You can add the brand here as a acomment:
```python
        MODELS_INFO: [
            ("_TZE200_zppcgbdj", "TS0601"),  # Conecto TH
```

**Review by javicalle — CHANGES_REQUESTED** [review]

**Comment by javicalle** [comment]
The quirk is fine in general. Just some minor comments.

**Comment by pianosaurus** [comment]
Agreed, and implemented as suggested.

**Review by javicalle — APPROVED** [review]
LGTM

**Review by TheJulianJES — APPROVED** [review]

**Pull request merged**